epidose/device/util.sh

```diff
@@ -238,7 +238,7 @@ log 'Starting up'
 
 # Obtain server URL
 if [ -n "$1" ] ; then
-  export SERVER_URL="$1"
+  export SERVER_URL="https://istlab.dmst.aueb.gr/epidose/"
```

Comment:
This breaks the ability to specify the server url as an argument to `util.sh`. You may want to set it afterwards as a default.  Also,  fix the instance `SERVER_URL="http://ha-server:5010" . "$UTIL"`  and don't forget the DRY principle

Reply:
Here is another way, I have tried to set SERVER_ULR as it was and instead changed the server URLs inside `epidose/epidose/device/supervisord.conf` to https://istlab.dmst.aueb.gr/epidose/. 
However, you need to build again the package for some reason (`make package && sudo make install`) which takes a lot of time.